pkgs/development/python-modules/dogpile.cache/default.nix

```diff
@@ -13,12 +13,12 @@ buildPythonPackage rec {
   };
 
   propagatedBuildInputs = [ dogpile_core ];
-  buildInputs = [ pytest ];
+  buildInputs = [ pytest pytestcov mock Mako ];
 
   meta = with stdenv.lib; {
     description = "A caching front-end based on the Dogpile lock";
     homepage = http://bitbucket.org/zzzeek/dogpile.cache;
-    platforms = platforms.linux; # Can only test linux
+    platforms = platforms.unix;
```

Comment:
Ah, that's why it wasn't there before. The previous update added `platforms.linux`